File: Cardiumm-cs50-2018-x-mario-less-20180815T161505Z/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int n;
    // Gets a positive number which is less than 23 from the user.
    do
    {
        n = get_int("Positive number between 1 and 23: ");
    }
    while (n < 0 || n > 23);

    // Loops until desired pyramid length
    for (int i = 0; i < n; i++)
    {
        // Drawing spaces to alin right.
        for (int j = 0; j < (n - i) - 1; j++)
        {
            printf(" ");
        }

        //Drawing pound signs. Using (i+1) to get 2 blocks on the first row.
        for (int r = 0; r <= (i + 1); r++)
        {
            printf("#");
        }
        // New line to give a proper shape.
        printf("\n");
    }

}

File: Cardiumm-cs50-2018-x-sentimental-mario-less-20180824T212826Z/mario.py
from cs50 import get_int

# Gets the user input.
while True:
    size = get_int("Size of the tower: ")
    if 0 <= size < 24:
        break

# Prints the tower
for i in range(size):
    i += 1
    print(" " * (size - i), end="")
    print("#" * (i + 1))